src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
 import './ChatEntry.css';
+
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const changeHeart = () => {
+    props.toggleHeartCallback(props.id);
+  };
```

Comment:
This is a good use of a helper function enclosing an outside value to have a simpler parameter list!